daemon/cmd/daemon_main.go

```diff
@@ -253,9 +253,17 @@ func init() {
 	flags.StringSlice(option.DebugVerbose, []string{}, "List of enabled verbose debug groups")
 	option.BindEnv(option.DebugVerbose)
 
-	flags.StringSliceP(option.Device, "d", []string{}, "List of devices facing cluster/external network for attaching bpf_netdev (first device should be one used for direct routing if tunneling is disabled)")
+	flags.StringP(option.Device, "d", "",
+		fmt.Sprintf("Device facing cluster/external network for attaching bpf_host. Deprecated in favor of --%s", option.Devices))
+	flags.MarkHidden(option.Device)
```

Comment:
also `flags.MarkDeprecated(option.Device, "This option is no longer supported and will be removed in v1.10")`

Reply:
It's supported, but it will be removed. Anyway, will add an description.